homeassistant/components/vesync/const.py

```diff
@@ -9,4 +9,25 @@
 VS_FANS = "fans"
 VS_LIGHTS = "lights"
 VS_SENSORS = "sensors"
+VS_HUMIDIFIERS = "humidifiers"
+VS_NUMBERS = "numbers"
 VS_MANAGER = "manager"
+
+DEV_TYPE_TO_HA = {
+    "LV-PUR131S": "fan",
+    "Core200S": "fan",
+    "Core300S": "fan",
+    "Core400S": "fan",
+    "Classic300S": "humidifier",
```

Comment:
I think you could also add "Dual200S": "humidifier", to this list

Reply:
While it looks like the master branch of pyvesync has Dual200S support, the PR for that was just merged two weeks ago (https://github.com/webdjoe/pyvesync/pull/104), so it's not in the latest release, which is from November. This should be possible after the next pyvesync release, though.

Reply:
I have found my Humidifier, ostensibly a Dual200S is reporting as "LUH-D301S-WEU" so it would be good to add that model as well.

From pyvsync:
```
{
	"Device Name": "Study Hum",
	"Model": "LUH-D301S-WEU",
	"Subdevice No": "None",
	"Status": "on",
	"Online": "online",
	"Type": "wifi-air",
	"CID": "vsaqxxxxxxxxxxxxxxxxxxxxxxx",
	"Mode": "auto",
	"Humidity": "40",
	"Mist Virtual Level": "1",
	"Mist Level": "1",
	"Water Lacks": false,
	"Humidity High": false,
	"Water Tank Lifted": false,
	"Display": true,
	"Automatic Stop Reach Target": true,
	"Auto Target Humidity": "42",
	"Automatic Stop": true
}
````